plugins/sample/plugin_sample.c

```diff
@@ -63,22 +63,22 @@ PLUGIN_EXPORT bool sir_plugin_query(sir_plugininfo* info) {
     info->desc      = desc;
     info->caps      = caps;
 
-    (void)printf("\t" DGRAY("plugin_sample ('%s')") "\n", __func__);
+    (void)printf("\t" SIR_DGRAY("plugin_sample ('%s')") "\n", __func__);
```

Comment:
technically, if we're going to have that CRLF setting, we should have a `SIR_LE` macro for line endings so we can clean these up too

Reply:
I can do that

Reply:
Done.